src/main/webapp/src/app/map/map.component.html

```diff
@@ -1,7 +1,7 @@
-<div style="text-align:center">
+
   <h1>
     Welcome to Maps!
   </h1>
-  <img src="map.jpg" alt="image">
+   <div class="image"><img src="map.jpg"/></div>
```

Comment:
Nit but potentially separating out div/img in to multi lines so it is slightly more readable.

<div class="image">
     <img src="map.jpg"/>
</div>

Reply:
fixed